geonotebook/layers.py

```diff
@@ -151,92 +217,86 @@ def next(self):
         except IndexError:
             raise StopIteration()
 
-# GeonotebookStack supports dict-like indexing on a list
-# of Geonotebook Layers. We could implement this with an
-# OrderedDict,  but i think we are eventually going to want
-# to support re-ordering,  potentially serializing etc so it
-# Seems like putting it in its own class is best for now.
 
-# TODO: support slices other list functionality etc
-# TODO: convert to collections.mutablesequence
 class GeonotebookStack(object):
```

Comment:
This wasn't added in this PR - but is this name appropriate? It's not technically a stack, and it's designed specifically to store Layer objects.